src/Sylius/Component/Customer/composer.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "sylius/customer",
+    "type": "library",
+    "description": "Customer handling for PHP applications.",
+    "keywords": ["customer", "groups"],
+    "homepage": "http://sylius.org",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Paweł Jędrzejewski",
+            "homepage": "http://pjedrzejewski.com"
+        },
+        {
+            "name": "Michał Marcinkowski",
+            "email": "michal.marcinkowski@lakion.com"
+        },
+        {
+            "name": "Łukasz Chruściel",
+            "email": "lukasz.chrusciel@lakion.com"
+        },
+        {
+            "name": "Sylius project",
+            "homepage": "http://sylius.org"
+        },
+        {
+            "name": "Community contributions",
+            "homepage": "http://github.com/Sylius/Sylius/contributors"
+        }
+    ],
+    "require": {
+        "php": "^5.6|^7.0",
+
+        "doctrine/orm": "^2.4.8,<2.5",
```

Comment:
For `Collection` we can simply add `doctrine/collections` package.

The method:
```
    /**
     * @return QueryBuilder
     */
    public function getFormQueryBuilder();
```
is no longer used, so we can remove the `GroupRepositoryInterface`.